pkg/collector/stats/utils.go

```diff
@@ -224,8 +224,15 @@ func getS390xArchitecture() (string, error) {
 func getCPUMicroArchitecture(family, model, stepping string) (string, error) {
 	yamlBytes, err := os.ReadFile(CPUModelDataPath)
 	if err != nil {
-		klog.Errorf("failed to read cpus.yaml: %v", err)
-		return "", err
+		currentFilePath, err := os.Getwd()
+		if err != nil {
+			return "", err
+		}
+		localCPUModelDataPath := strings.SplitAfter(currentFilePath, "kepler")[0]
+		yamlBytes, err = os.ReadFile(localCPUModelDataPath + "/data/cpus.yaml")
+		if err != nil {
+			return "", err
+		}
 	}
```

Comment:
@gdozortsev I tested this locally and works fine 
```
func readCPUModelData() ([]byte, error) {
	data, err := os.ReadFile(CPUModelDataPath)
	if err == nil {
		return data, nil
	}
	if !os.IsNotExist(err) {
		return nil, err
	}

	klog.Infof("reading local cpus.yaml")
	return os.ReadFile("./data/cpus.yaml")
}

func getCPUMicroArchitecture(family, model, stepping string) (string, error) {
	yamlBytes, err := readCPUModelData()
```

Tested using ... 
```
make build && sudo ./_output/bin/kepler 2>&1
```

Reply:
@gdozortsev could you please try if this works - https://github.com/sustainable-computing-io/kepler/pull/1572/files